modules/caddyhttp/reverseproxy/httptransport.go

```diff
@@ -546,6 +552,7 @@ func (t TLSConfig) MakeTLSClientConfig(ctx caddy.Context) (*tls.Config, error) {
 
 	// trusted root CAs
 	if len(t.RootCAPool) > 0 || len(t.RootCAPEMFiles) > 0 {
+		ctx.Logger().Warn("RootCAPool and RootCAPEMFiles are deprecated. Use one of the tls.ca_pool.source modules instead")
```

Comment:
Same for the fields names here. The users only know `root_ca_pool` and `root_ca_pem_files`.